recipe/meta.yaml

```diff
@@ -20,17 +20,23 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
     - {{ stdlib("c") }}
     - cmake
     - ninja        # [unix]
+    - make         # [unix]
```

Comment:
@SwayamInSync I was wondering why is ``make`` needed here, as the recipe on Unix uses ``ninja`` as generator?

Reply:
Thanks for catching that, make sense, I would've just put that there for my local use, will push a refactor commit for this